### ISSUE TITLE
Amay Advani Reflection Review Ticket

### ISSUE THREAD

**Issue by amayadvani** [amayadvani/fastpages#26]
## Skills I have obtained

* Independent Work: This course has been primarily individual, meaning mostly students working together and less involvement with the teacher. This art of independence has helped me keep my self accountable with completeing my own work, and even remind others to finish their own tasks. I have implemented this into more subjects in my life other then computer science and has been one of the best things I have taken away from this class.
* Teaching/explaining your code: In the several lesson presentations in which you and your group were assigned to choose a topic and present a lesson to the class with good class interactions and hacks gave me valuable experience. Additionally, most of the live reviews, I would have to either explain the code that I wrote and how it connects to the assigned hacks, almost to the point of detail to which someone else could recreate your whole feature.
* Collaboration: Before taking this class, I knew what collaboration was, but it had a different definition. This class taugh me the difference between collaboration in the workplace versus collaboration in school. In a "workplace" each person must think about the goals of the whole team rather then their own individual success. The chemistry and art of working with one another is the greatest in csp, and that isn't due to the people I was with, but rather the way the class was formated. 

## Highs and Lows

* Some lows that I had were in the beginning of each trimester. First trimester, it was a pain to get vscode connected with github and have it working flawlessly, even though many unseen problems had popped up. In second trimester, I wasn't expecting the jump in expectation and work it took to complete my hacks. This resulted in many sleepless nights finishing up code of confusing topics that was half chat gpted. Although, overtime this workload had become more manageable and even fun.
* Another low that had affected me personally, was my missing days of school due to my traveling for robotics. It was hard to bounce back from not learning these new concepts, especially because I had to catch up with everything else from all my other classes as well. 
* One high from this year was finally getting my frontend connected with my backend during second tri. I remeber in first tri everything was pretty straigh forward and we werent sending any crud requests for the api, but in second tri I had encounted several errors that just had no explaination, athough, me finally fiquring them out and getting the frontend and backend to connect was one of the most rewarding and relaxing moments in my csp year. 

## Future plans with cs +education

* College: As I am still deciding what I want to minor in, it it definitly boiling down to business or computer science. This class definitly opened my eye to the world of cs and what is expected in college. But I would like to continue using vscode and finishing up some passion projects over the summer. 

## Would I recomend this to a new student

* One major thing that I took away from this class was going above and beyond. In other classes, to get full credit, the parameters are clearly stated, but in this class the requirements are all relative, which makes the extra credit very creative. I've learned from this class that performing the bare minimum and acting simply are insufficient in the reqirements of this class, and in overall society. By going above and above, you may inspire people around you to do even better. Going above and above teaches you to understand the project you are working on better.

* In conclusion, I would reccomend this class to a new student, not only because you become more proficent in cs, but also because of the life skills that yoi learn with taking this class. Thank You Mr. Mortenson.
